### PR TITLE
D8CORE-5255 D8CORE-5256: lockup and templating work

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Black banner bar
- Black footer bar
- Fix up the logo
- Remove the copyright
- Removed the Stanford wordmark with display: none. 

# Review By (Date)
- ASAP

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to homepage
4. Verify 
- Black banner bar
- Black footer bar
- Fix up the logo - removed the lines and kept padding between logo uploaded and lines added.
- Remove the copyright
- Removed the Stanford wordmark with display: none. If no logo is uploaded, there is no Stanford wordmark. This might need another approach but I want to get the bulk of these changes merged in.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? NO

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-5257
- https://stanfordits.atlassian.net/browse/D8CORE-5256
- https://stanfordits.atlassian.net/browse/D8CORE-5255

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: templates/components/brand-bar/brand-bar.html.twig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *   - Dark: White wordmark over dark grey background
+ *   - White: Cardinal red wordmark over white background
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ */
+#}
+
+<div class="su-brand-bar su-brand-bar--dark {{ modifier_class }}">
```
Comment (jdwjdwjdw) [inline templates/components/brand-bar/brand-bar.html.twig]: I'm seeing an additional unnecessary `su-brand-bar--` added to the html when I inspect. I think if you remove `{{ modifier_class }}` that should fix that.
Screen Shot 2022-01-27 at 4 19 52 PM

Somewhat related brand bar/global footer question - do we want those two components to have the same height?

**Review by jdwjdwjdw — COMMENTED** [review]
Looking good! Just the brand bar questions, then we should be good to merge.

File: templates/components/brand-bar/brand-bar.html.twig (hunk 1 above)
Comment (jenbreese) [reply templates/components/brand-bar/brand-bar.html.twig]: Yes you are correct they should be the same and 30px according to figma. I fixed those in the fixups.

**Review by jdwjdwjdw — APPROVED** [review]
Good to go!

**Pull request merged**